libs/vscode/nx-project-view/src/lib/nx-project-tree-view.ts

```diff
@@ -0,0 +1,177 @@
+import { ProjectConfiguration } from '@nrwl/devkit';
+import { CliTaskProvider } from '@nx-console/vscode/tasks';
+import { getOutputChannel } from '@nx-console/vscode/utils';
+import { TreeItemCollapsibleState } from 'vscode';
+import { AbstractView, isDefined, TreeViewStrategy } from './nx-project-helper';
+import {
+  NxFolderTreeItem,
+  NxProjectTreeItem,
+  NxTreeViewItem,
+} from './nx-project-tree-item';
+
+type TreeViewMap = Map<string, [string, ProjectConfiguration][]>;
+
+export function createTreeViewStrategy(
+  cliTaskProvider: CliTaskProvider
+): TreeViewStrategy {
+  const listView = new TreeView(cliTaskProvider);
+  return {
+    getChildren: listView.getChildren.bind(listView),
+    getParent: listView.getParent.bind(listView),
+  };
+}
+
+class TreeView extends AbstractView {
+  constructor(cliTaskProvider: CliTaskProvider) {
+    super(cliTaskProvider);
+  }
+  async getParent(element: NxTreeViewItem) {
+    if (element instanceof NxFolderTreeItem) {
+      if (this.isRootPath(element.path)) {
+        return null;
+      }
+      const projectDefs = await this.cliTaskProvider.getProjects();
+      const map = this.groupByRootPath(projectDefs);
+      const parentFolders = this.getParentFolder(map, element.path);
+      return parentFolders.map(([path]) => this.createFolderTreeItem(path));
+    }
+
+    if (element instanceof NxProjectTreeItem) {
+      const projectRoot = element.nxProject.root;
+      const projectDefs = await this.cliTaskProvider.getProjects();
+      const map = this.groupByRootPath(projectDefs);
+      const parentFolders = this.getParentFolder(map, projectRoot);
+      return parentFolders.map(([path]) => this.createFolderTreeItem(path));
+    }
+
+    return this.getParentOfTargetItem(element);
+  }
+
+  async getChildren(element?: NxTreeViewItem) {
+    if (!element) {
+      return this.createFolders();
+    }
+    if (element instanceof NxFolderTreeItem) {
+      return this.createFolders(element);
```

Comment:
Currently folders have a `Edit Workspace Definition` button which doesn't do anything. Please remove those (but maybe a 'show in explorer' button is useful). 

<img width="453" alt="image" src="https://user-images.githubusercontent.com/34165455/197306184-8078708a-9eff-4c04-9936-e9b739c5f66e.png">

Reply:
Fixed, for folders it now shows `Reveal in Explorer`